composer.json

```diff
@@ -29,10 +35,9 @@
     },
     "require": {
         "php": "^7.2|^8.0",
-        "nunomaduro/larastan": "^0.6.11|^0.7",
+        "nunomaduro/larastan": "^1.0.0",
         "nikic/php-parser": "^4.0",
         "laravel/framework": "^6.0|^7.0|^8.0",
```

Comment:
Do we need Laravel as an actual dependency here or should this be a dev dependency?

Reply:
I believe this should stay as a main dependency as we consume it as a dev dependency in projects.

Reply:
What I mean sorry is that the code standards shouldn't need Laravel as a dependency, more as a development dependency. Otherwise, our Laravel app has a dependency for Laravel by proxy.

It doesn't change much as this is a Dev dependency itself, just one tiny thing 🙂

Reply:
actually isnt it needed to know what versions of laravel to install this on? so you wouldnt want it to try and install on a laravel 5 version for instance.

Reply:
im going to leave this as is for now we can look at this in more detail in the next PR